src/capture/inactive.rs

```diff
@@ -98,6 +98,17 @@ impl Capture<Inactive> {
         self
     }
 
+
+    /// Set want_pktap to true or false. The default is maintained by pcap.
```

Comment:
The default is maintained by **libpcap**. We use a naming convention where `libpcap` refers to the underlying library and `pcap` refers to this crate.

Reply:
Thanks for catching that, updated the comment :)